yarn.lock

```diff
@@ -3046,13 +2986,6 @@
   dependencies:
     core-js "^3.0.1"
 
-"@storybook/core-events@5.2.8":
```

Comment:
Were these all dependencies of the addon centered plugin? Seems like a lot

Reply:
that came with `yarn remove @storybook/addon-centered`, most of them are different version of what's imported/cross-referenced by the other storybook plugins
should I just manually remove the dependency on package.json instead?

Reply:
Nope, sounds good. Didn't realize the version difference